contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -92,23 +109,31 @@
                         <item value="UV Plasma Source"/>
                         <item value="Metal Jet X-ray"/>
                         <item value="HHG laser"/>
+                        <item value="UV lamp"/>
+                        <item value="Monochromatized electron source"/>
+                        <item value="other"/>
                     </enumeration>
                 </field>
-                <field name="name"/>
+                <field name="type_other" optional="true">
+                    <doc>
+                        Specification of type, may also go to name.
+                    </doc>
+                </field>
+                <field name="name" recommended="true"/>
                 <field name="probe">
                     <doc>
                          Type of probe. In photoemission it's always photons, so the full NIAC list is
                          restricted.
                     </doc>
                     <enumeration>
-                        <item value="x-ray"/>
-                        <item value="ultraviolet"/>
-                        <item value="visible light"/>
+                        <item value="photons"/>
                     </enumeration>
                 </field>
             </group>
-            <group type="NXbeam">
-                <field name="distance" type="NX_NUMBER" units="NX_LENGTH">
+            <group name="beam_probe" type="NXbeam">
+                <!-- Comment from mpes may workshop: Add linking between source and beam-->
+                <!-- Add extent as recommended?-->
+                <field name="distance" type="NX_NUMBER" units="NX_LENGTH" recommended="true">
                     <doc>
                          Distance of the point of evaluation of the beam from the sample surface.
```

Comment:
Docstring is hard to understand and should be clarified.

Reply:
NeXus gives the shorter, but more ambiguous docstring "Distance from sample". NXbeam evaluates the beam properties at a point along the beam path, so this is the "Distance between the point where the current NXbeam instance is evaluating the beam properties and the point where the beam interacts with the sample. For photoemission, the latter is the point where the the centre of the beam touches the sample surface."

Reply:
We are using Tommaso's new, so I am resolving here.